src/packages/core/workspace/workspace-property-dataset/invariant-workspace-property-dataset-context.ts

```diff
@@ -0,0 +1,65 @@
+import {
+	UMB_PROPERTY_DATASET_CONTEXT,
+	UmbPropertyDatasetContext,
+	UmbNameablePropertyDatasetContext,
+} from '@umbraco-cms/backoffice/property';
+import { type UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import { UmbBaseController } from '@umbraco-cms/backoffice/class-api';
+import { UmbVariantId } from '@umbraco-cms/backoffice/variant';
+import { UmbInvariantableWorkspaceContextInterface } from '@umbraco-cms/backoffice/workspace';
+
+/**
+ * A property dataset context that hooks directly into the workspace context.
+ */
+export class UmbInvariantWorkspacePropertyDatasetContext<
+		WorkspaceType extends UmbInvariantableWorkspaceContextInterface = UmbInvariantableWorkspaceContextInterface,
+	>
+	extends UmbBaseController
+	implements UmbPropertyDatasetContext, UmbNameablePropertyDatasetContext
+{
+	#workspace: WorkspaceType;
+
+	name;
+
+	// default data:
+
+	getVariantId() {
+		return UmbVariantId.CreateInvariant();
+	}
+	getType() {
```

Comment:
I have started to use the name `entityType` explicitly instead of just `type`. Would it be ok to do the same here so the prop is `getEntityType`? I have noticed it has removed some confusion in other places in the code because `type` is used a lot for many different things.

Reply:
Yeah cool. And good perspective.  This is just transferred from the existing interface, so i'm all in for changing this.